sefaria/urls.py

```diff
@@ -13,6 +13,7 @@
 import sourcesheets.views as sheets_views
 import sefaria.gauth.views as gauth_views
 import django.contrib.auth.views as django_auth_views
+import api.views as v3_views
```

Comment:
I think this is going to get confusing later on. We really should not be naming things inside the code as "v3". Rather I suggest something more sustainable and straightforward like "api_views"

Reply:
why not v3? it's officialy v3, so I think we can name it like this. on the other hand, it's not the only api views we'll have, so api_views seems confusing to me. as long as we use api from more than one version I think naming it after its version is the most informative.